crates/common/src/version.rs

```diff
@@ -0,0 +1,11 @@
+// The version of the Foundry CLI.
+pub const VERSION_MESSAGE: &str = env!("FOUNDRY_SHORT_VERSION");
```

Comment:
Let's have 2 different constants, one for the semver compatible version that we use in the cheatcode and one that's the --version output.

I think the --version message should be formatted more nicely than a compact semver version, see for example reth and [solar](https://github.com/paradigmxyz/solar/blob/0858f56149709cc6cbcf4b298d229d64ec99ef1a/crates/config/build.rs)

Reply:
Added, now there are two different versions:

Semver compatible for the cheatcode

````
forge 0.3.0-stable+b3d0002118.1737037945.release
```

and the message

```
forge 
Version: 0.3.0-nightly
Commit SHA: d22d7579923463cfe84fa46d073f4efe1ebd1e22
Build Timestamp: 2025-01-16T14:21:42.620482424Z
Build Profile: debug
```

Regarding `Build Features`, there appears to be a requirement to enable flags for Vergen to pick it up correctly . I don't think it is necessary to include it in this version message.

https://github.com/paradigmxyz/reth/blob/590496d0e42f0c17fa54cd11db177b7cee3e6425/crates/node/core/Cargo.toml#L80-L82

Reply:
yeah that's fine